### PR TITLE
Minor changes for gitignore parsing

### PR DESCRIPTION
**Pull request by michaelkedar**
@oliverchang pointed out a few things #206 after it was merged

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -32,10 +32,10 @@ Git directories are searched for the latest commit hash. Searching for git commi
 
 ### Ignored files
 
-By default, OSV-Scanner will not scan files that are ignored by `.gitignore` files. If the specified file is not part of a git repository, `.gitignore` files are parsed recursively starting from the target directory,
-otherwise they are parsed from the root of the git repository as typical.
+By default, OSV-Scanner will not scan files that are ignored by `.gitignore` files. If the specified file is not part of a git repository, any `.gitignore` files in the target directory and all subdirectories will be parsed,
```
Comment (oliverchang) [inline USAGE.md:35]: Can you update this to reflect our latest discussion in https://github.com/google/osv-scanner/pull/206#discussion_r1103972400 ?

File: USAGE.md [hunk 2]
```diff
@@ -32,10 +32,9 @@ Git directories are searched for the latest commit hash. Searching for git commi
 
 ### Ignored files
 
-By default, OSV-Scanner will not scan files that are ignored by `.gitignore` files. If the specified file is not part of a git repository, `.gitignore` files are parsed recursively starting from the target directory,
-otherwise they are parsed from the root of the git repository as typical.
+By default, OSV-Scanner will not scan files that are ignored by `.gitignore` files. All recursively scanned files are matched to a git repository (if it exists) and any matching `.gitignore` files within that repository are taken into account.
```
Comment (michaelkedar) [inline USAGE.md:35]: I don't know if something about #209 is worth mentioning here

**Comment by michaelkedar** [comment]
FYI this PR modifies `USAGE.md` again #201 #203

File: USAGE.md (hunk 2 above)
Comment (oliverchang) [reply USAGE.md:35]: Let's just link it to be clear to people about known bugs.

**Review by oliverchang — APPROVED** [review]
LGTM with just one comment.

File: USAGE.md (hunk 2 above)
Comment (michaelkedar) [reply USAGE.md:35]: Added a link

**Review by oliverchang — APPROVED** [review]
LGTM

**Pull request merged**